OSDOCS-13550 hardcoded 4.19 version number bump (#93965)
openshift/openshift-docs: modules/oc-mirror-imageset-config-params.adoc

|`mirror.operators.catalog`
|The Operator catalog to include in the image set.

> |String. For example: `registry.redhat.io/redhat/redhat-operator-index:v4.19`.

Reviewer: ```suggestion
|String. For example: `registry.redhat.io/redhat/redhat-operator-index:v{product-version}`.
```

|`mirror.operators.packages.channels.name`
|The Operator channel name, unique within a package, to include in the image set.

> |String. For example: `fast` or `stable-v4.19`.

Reviewer: ```suggestion
|String. For example: `fast` or `stable-v{product-version}`.
```

|`mirror.platform.channels.name`
|The name of the release channel.

> |String. For example: `stable-4.19`

Reviewer: ```suggestion
|String. For example: `stable-{product-version}`
```

|`mirror.platform.channels.minVersion`
|The minimum version of the referenced platform to be mirrored.
|String. For example: `4.12.6`

|`mirror.platform.channels.maxVersion`
|The highest version of the referenced platform to be mirrored.

> |String. For example: `4.19.1`

Reviewer: ```suggestion
|String. For example: `{product-version}.1`
```
